great_expectations/core/usage_statistics/anonymizers/base.py

```diff
@@ -72,8 +72,10 @@ def get_parent_class(
                 object_module_name = object_config.get("module_name")
                 object_class = load_class(object_class_name, object_module_name)
 
-            object_class_name = object_class.__name__
-            object_module_name = object_class.__module__
+            # TODO: <Alex>ALEX</Alex>
+            # object_class_name = object_class.__name__
+            # object_module_name = object_class.__module__
+            # TODO: <Alex>ALEX</Alex>
```

Comment:
What's the purpose of this? I'd rather we completely delete code we don't need or provide some additional context.